jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+version: 2.1
+
+jobs:
+  sayhello:
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - say:
+          # Since the command "say" doesn't define a default
+          # value for the "saywhat" parameter, it must be
+          # passed in manually
+          saywhat: << parameters.saywhat >>
+
+commands:
+  say:
+    parameters:
+      saywhat:
+        type: string
+    steps:
+      - run: echo "<< parameters.saywhat >>"
+
+workflows:
+  build:
+    jobs:
+      - sayhello:
+          saywhat: Everyone
+```
+
+
+
+### Invoking the Same Job Multiple Times
+{:.no_toc}
+
+A single configuration may invoke a job many times. At configuration processing time during build ingestion, CircleCI will auto-generate names if none are provided.  If you care about the name of the duplicate jobs, they can be explicitly named with the `name` key.
+
+**Note:** The user must explicitly name repeat jobs when a repeat job should be upstream of another job in a workflow (ie: if the job is used under the `requires` key of a job invocation in a workflow you will need to name it).
+
+```yaml
+version: 2.1
+workflows:
+  build:
+    jobs:
+      - loadsay
+      # This doesn't need an explicit name as it has no downstream dependencies
+      - sayhello:
+          saywhat: Everyone
+          requires:
+            - loadsay
+      # This needs an explicit name for saygoodbye to require it as a job dependency
+      - sayhello:
+          name: SayHelloChad
+          saywhat: Chad
+      # Uses explicitly defined "sayhello"
+      - saygoodbye:
+          requires:
+            - SayHelloChad
+```
+
+### Pre and Post Steps
+{:.no_toc}
+
+**Note:** The keys `pre-steps` and `post-steps` in jobs are available in configuration version 2.1 and later_
+
+Every job accepts two special arguments: `pre-steps` and `post-steps`.
+You can optionally invoke a job with one or both of these arguments. Steps under `pre-steps`
+are executed before any of the other steps in the job. The steps under
+`post-steps` are executed after all of the other steps.
+
+For this reason, the parameter names `pre-steps` and `post-steps` are reserved
+and may not be redefined by a job author. For example, the following job
+definition is invalid:
+
+```yaml
+jobs:
+  foo:
+    parameters:
+      pre-steps:    # invalid: pre-steps is a reserved parameter name
+        type: steps
+        default: []
+```
+
+All jobs accept two special arguments of type `steps`: `pre-steps` and
+`post-steps`.
+
+If an orb user invokes a job with one or both of these arguments,
+the job will run the steps in `pre-steps` first, before any other steps run, and
+then it will run the steps in `post-steps` last, after any other steps run.
+
+Pre- and post- steps allow users to be execute steps in a given job's environment
+without modifying the orb. This is useful, for example, when a user imports a job
```

Comment:
Use of the word "orb" here isn't necessary -- replace with "job"

Reply:
replaced, thanks